src/App.tsx

```diff
@@ -1,14 +1,50 @@
 import "@buildo/bento-design-system/index.css";
 import "@buildo/bento-design-system/defaultTheme.css";
 import { defaultMessages } from "@buildo/bento-design-system/defaultMessages/en";
-import { BentoProvider, Title } from "@buildo/bento-design-system";
+import {
+  BentoProvider,
+  Headline,
+  Box,
+  ContentWithSidebar,
+  Body,
+} from "@buildo/bento-design-system";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import './i18n';
+import { useTranslation } from "react-i18next";
+import Home from "./pages/Home";
+
+
 
 function App() {
+  const { t } = useTranslation();
   return (
     <BentoProvider defaultMessages={defaultMessages}>
-      <Title size="large">Hello, World!</Title>
+      <BrowserRouter>
+        <Headline size="large" align="center">
+          {t("title")}
+        </Headline>
```

Comment:
I would add a bit of padding around this title, and maybe a colored background to better separate this heading from the page content.